### PR TITLE
Updated changelogs.

### PR DESCRIPTION
**Pull request by vinjana**

### CODE REVIEW

**Comment by vinjana** [comment]
@NagaComBio 

I proposed to add the following release branches and tags on these branches for the backport of the bugfix:

 * Add a ReleaseBranch_2.4.1 on which to tag 2.4.1-1
 * Add a ReleaseBranch_2.2 on which to tag 2.2.2; the branch already exists with few commits
 * Add a ReleaseBranch_2.0.0 on which to tag 2.0.0-101. This branches off 2.0.0-1 which only adds readme updates to the tag 2.0.0. Note that tag 2.0.0-2 already has content changes that wouldn't take into this branch. Therefore the release number 2.0.0-101.
 * Add a ReleaseBranch_1.2.177-6 on which to tag 1.2.177-601.

Furthermore, I propose to add two new tags

 * 3.0.0 on what is 2.5.0 now, and rename 2.5.0 to 2.5.0-deprecated
 * 3.1.0 on what is 2.6.0 now, and rename 2.6.0 to 2.6.0-deprecated

Thus the numbers will express the compatibility.

I added all this to the changelog.

Do you agree with these changes?

File: README.md [hunk 1]
```diff
@@ -138,13 +166,39 @@ TBD
...
-  - 1.2.177 is equivalent to 1.0.176-9.
-  - 1.2.177-1 is equivalent to 1.0.176-10.
+* Version update to 1.2.177-7
 
-* Version update to 1.0.176-9
+* Version update to 1.2.177-601 (ReleaseBranch_1.2.177-6)
+
+  - Bugfix: Fix of column swap bug introduced before 1.0.167. Output VCF with swapped control and tumor genotype columns if they are in the 11th and 10th column respectively.
+
+* Version update to 1.2.177-6
```
Comment (NagaComBio) [inline README.md:177]: These changes are in 1.2.177-5

Comment (vinjana) [reply README.md:177]: The bugfix is what you are just about to create, right? And it is what is described here. There is a ReleaseBranch_1.2.177-6, that actually also starts with 1.2.177-6, not with the previous tag.

So in the moment I don't understand your comment.

Comment (vinjana) [reply README.md:177]: Ah, right, I see what you mean.

**Review by NagaComBio — APPROVED** [review]

**Pull request merged**